lib/MqttClient.js

```diff
@@ -13,8 +13,11 @@ const MQTT_COMMANDS = {
 };
 
 const CUSTOM_COMMANDS = {
+    APP_GOTO_TARGET: "app_goto_target",
+    APP_SEGMENT_CLEAN: "app_segment_clean",
```

Comment:
APP_SEGMENT_CLEAN is not used in your code.
Valetudo currently does not support segment cleaning.

Reply:
Yeah, this obviously true, my bad. I realized that, then unfortunately forgot about this.

How about keeping it in there, but adding a case logging a message saying it is not supported yet. This way it would be indicated that it is part of the api, generally supported by other software, but not implemented with Valetudo (maybe yet).